garrysmod/lua/weapons/weapon_fists.lua

```diff
@@ -18,216 +18,303 @@ SWEP.UseHands = true
 SWEP.Primary.ClipSize = -1
 SWEP.Primary.DefaultClip = -1
 SWEP.Primary.Automatic = true
-SWEP.Primary.Ammo = "none"
+SWEP.Primary.Ammo = ""
 
 SWEP.Secondary.ClipSize = -1
 SWEP.Secondary.DefaultClip = -1
 SWEP.Secondary.Automatic = true
-SWEP.Secondary.Ammo = "none"
+SWEP.Secondary.Ammo = ""
 
 SWEP.DrawAmmo = false
 
-SWEP.HitDistance = 48
-
-local SwingSound = Sound( "WeaponFrag.Throw" )
-local HitSound = Sound( "Flesh.ImpactHard" )
+SWEP.HoldType = "fist"

```

Comment:
I don't think this is useful. Addons can and should use `Weapon.GetHoldType`, and I don't see any reason to have fists swep use not fist hold type.

Reply:
It's consistent with what was done with the medkit and is a pseudo-standard for SWEPs out in the wild. I'm not going to theorize why someone would change the fists holdtype but it's there. Built-in SWEPs should be an example for new coders and consistency is how you show something is intentional good practice rather than it being something unique to the SWEP/a mistake.